deps/wazuh_testing/wazuh_testing/tools/macos_log/macos_utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
This new module should be placed in the logcollector module  `/deps/wazuh_testing/wazuh_testing/logcollector.py`. The folder `macos_log`  should be used only for non-python scripts used in macOS logs generation.